generate.py

```diff
@@ -1,10 +1,12 @@
 import sqlite3
+import psycopg2
```

Comment:
since, we're adding new dependencies, can you please add the dependencies and its version on requirements.txt

https://github.com/Lumenaut-Network/pool/blob/master/requirements.txt